content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -8,9 +8,10 @@ title: Debug Services
 
 {{% capture overview %}}
 An issue that comes up rather frequently for new installations of Kubernetes is
-that a `Service` is not working properly.  You've run your `Deployment` and
-created a `Service`, but you get no response when you try to access it.
-This document will hopefully help you to figure out what's going wrong.
+that a `Service` is not working properly.  You've run your `Pods` through a
+`Deployment` (or other workload controller) and created a `Service`, but you
+get no response when you try to access it.  This document will hopefully help
+you to figure out what's going wrong.
```

Comment:
@thockin: if you're in the mood to clean up, resource names like Pod, Deployment and Service should not be in backticks.

Reply:
Can do